.github/workflows/deploy_dev_gcp.yml

```diff
@@ -0,0 +1,61 @@
+name: Deploy to Dev environment.
+on:
+  push:
+    branches:
+      - main
+
+env:
+  PROJECT_DEV: "pagoda-discovery-platform-dev"
+  REGION: "us-central1"
+  IMAGE: "us-central1-docker.pkg.dev/pagoda-discovery-platform-dev/cloud-run-source-deploy/notification-server"
+  PR_NUMBER: ${{ github.event.number }}
+
+jobs:
+  build-notifications-server:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+        name: "Checkout notifications-server"
+
+      - name: Login to Artifact Registry
+        uses: docker/login-action@v2
+        with:
+          registry: ${{ env.REGION }}-docker.pkg.dev
+          username: _json_key
+          password: ${{ secrets.NOTIFICATION_SA_DEV_KEY }}
+
+      - name: Build Docker image and push to Google Artifact Registry
+        id: docker-push-tagged
+        uses: docker/build-push-action@v4
+        with:
+          push: true
+          file: ./Dockerfile
+          tags: "${{ env.IMAGE }}:${{ github.sha }}"
```

Comment:
I am not sure if I understand the tag here. I used this tag when deploying locally: _us-central1-docker.pkg.dev/pagoda-discovery-platform-dev/cloud-run-source-deploy/notification-server:latest_. Is this done differently from GH actions?